IGHTMLQuery/Classes/IGXMLNodeSet.h

```diff
@@ -13,16 +13,16 @@
 
 @class IGXMLNode;
 
-typedef void (^IGXMLNodeSetEnumerateBlock)(IGXMLNode* node, NSUInteger idx, BOOL *stop);
+typedef void (^IGXMLNodeSetEnumerateBlock)(IGXMLNode* _Nonnull node, NSUInteger idx, BOOL* _Nullable stop);
 
-typedef void (^IGXMLNodeSetEachBlock)(IGXMLNode* node);
+typedef void (^IGXMLNodeSetEachBlock)(IGXMLNode* _Nonnull node);
 
 @protocol IGXMLNodeSetCore <IGHTMLQueryJavaScriptExport>
 
 /**
  nodes in this node set
  */
-@property (nonatomic, copy, readonly) NSOrderedSet* nodes;
+@property (nullable, nonatomic, copy, readonly) NSOrderedSet* nodes;
```

Comment:
this seems better to be nonnull, some minor change in the implementation

``
        if (nodes) {
            _nodes = [NSOrderedSet orderedSetWithArray:nodes];
        } else {
            _nodes = [NSOrderedSet orderedSet];
        }
``